docs/infrastructure/feature branches jenkins.md

```diff
@@ -0,0 +1,50 @@
+# Jenkins in Feature Branches
```

Comment:
Should this file live in the wiki instead? Seems odd that this would get branched and merged with releases.

Reply:
WIKIS start at -1000 for me. They can't be contributed to by the community and have no realistic review process (can't PR it).